pure.zsh

```diff
@@ -598,7 +619,7 @@ prompt_pure_setup() {
 	PROMPT='%(12V.%F{242}%12v%f .)'
```

Comment:
Also missed a color here. Maybe `%F{${prompt_pure_colors[venv]}}` or `[virtualenv]`?

Reply:
It was intentional, nobody asked for it before and since the author want the project do be KISS I skipped it.